docs/tutorials/setting-up-python-pycharm.md

```diff
@@ -23,10 +23,11 @@ this up, head over to our [SSH tutorial](/docs/tutorials/connecting-to-ev3dev-wi
 ## Configure code synchronization
 
 {% include icon.html type="info" %}
-The first half of this tutorial assumes you are using the free version of PyCharm.
-However, if you have access to the Professional Version of PyCharm, which is free for students
-and some open-source projects, you should probably read the PyCharm Professional version
-alternatives shown later.
+The first half of this tutorial assumes you are using the free "Community"
+version of PyCharm. However, if you have access to the "Professional" version of
+PyCharm, which is free for students and some key open-source projects, you we
```

Comment:
Whatever the specifics in wording, I don't want to deal with the intricacies of their licensing scheme, so I've added a more generic phrase with a link instead.

> In fact, this statement is wrong. It is free to a limited number key contributors of verified open source projects (for use on said open source project, not for general use). 

It isn't wrong, although it could be elaborated on. JetBrains' requirements are specific on the types of projects that qualify -- open-source, in active development, _and_ running for more than three months -- and as a result the majority of readers would be disqualified by the project, rather than their role. Most users on GitHub have a couple tiny repos which they are "leads" in, but those projects don't meet activity requirements.